secrets/ingress_users.enc.yaml

```diff
@@ -1,117 +0,0 @@
-ingress_users:
```

Comment:
how come this got removed?

Reply:
maybe i made a mistake there? unsure

Reply:
without this, you won't be able to push to grafana.